.gitignore

```diff
@@ -85,4 +85,5 @@ tree-sitter-usfm3/src/
 js-usfm-parser/src/web-tree-sitter/tree-sitter.js
 js-usfm-parser/src/web-tree-sitter/tree-sitter.wasm
 rust-usfm-parser/target
-**target
\ No newline at end of file
+**target
+*/src/input.usfm/*
```

Comment:
This is a single file, right? we would not need the tailing slash and *. `rust-usfm-parser/src/input.usfm` should be the entry.